pages/aboutscript.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] Kindly ensure that your linters pass by showing the green check, avoid disabling linters, it would be better to remove this line 👍🏿